src/components/Application/Application.jsx

```diff
@@ -35,4 +65,12 @@ const mapStateToProps = state => ({
   settingsOpen: state.ui.settingsOpen,
 });
 
-export default connect(mapStateToProps, null)(Application);
+const mapDispatchToProps = dispatch => ({
+  updateConfig: config => dispatch(setConfig(config)),
```

Comment:
preference thing : you could write this as:
```
const mapDispatchToProps = dispatch => ({
    setConfig: config => dispatch(setConfig(config)
})
```

so basically what this is doing is, giving you your original actionCreator wrapped in the dispatch call (I believe)... so the equivalent of inlining the whole thing is:

```
ipcRenderer.on('config', props.dispatch({
  type: 'SET_CONFIG'
  payload: myPayloadObject 
});
```

or in the case of the fact that you are indeed (ias in this case) already wrapping the action creator you could just do:


```
ipcRenderer.on('config', props.dispatch(setConfig());
```
I think...